src/commands/dump.rs

```diff
@@ -1,11 +1,12 @@
 use std::path::{Path, PathBuf};
 
 use anyhow::Context;
+use indicatif::{ProgressBar, HumanBytes, ProgressStyle};
```

Comment:
Like the warning says, `ProgressStyle` is an unused import.

Reply:
Weird, rust-analyzer is blanking on me with the unused warnings. Thanks!